miqcli/collections/automation_requests.py

```diff
@@ -79,6 +79,19 @@ def create(self, method, payload, payload_file,):
                     tenant.get_id(input_data['tenant'])
 
             _payload = OSP_FIP_PAYLOAD
+        elif method == SUPPORTED_AUTOMATE_REQUESTS[2]:
```

Comment:
I'm not sure this strategy is a good option. If we change the order in SUPPORTED_AUTOMATE_REQUESTS we mess everything. 

It would be better if we could write something like `methods.RELEASE_FLOATING_IP`, `methods.GENERIC`, etc. We have to think more about this.

I'm OK to accept this to get the first phase accomplished, but it is definitely something that should be changed to a more pythonic style.

Reply:
I was following the model already created, but I agree I think we should improve this.